api/app/tests/apps/sigmax/test_soap.py

```diff
@@ -0,0 +1,190 @@
+import uuid
+from unittest import mock
+
+import lxml
+from django.http import HttpResponse
+from django.template.loader import render_to_string
+from django.test import TestCase
+from rest_framework.test import APITestCase
+
+from signals.apps.sigmax.views import (
+    ACTUALISEER_ZAAK_STATUS_SOAPACTION,
+    _parse_actualiseerZaakstatus_Lk01
+)
+from signals.apps.signals import workflow
+from signals.apps.signals.models import Signal
+from tests.apps.signals.factories import SignalFactoryValidLocation
+from tests.apps.users.factories import SuperUserFactory
+
+REQUIRED_ENV = {'SIGMAX_AUTH_TOKEN': 'TEST', 'SIGMAX_SERVER': 'https://example.com'}
+SOAP_ENDPOINT = '/signals/sigmax/soap'
+
+
+class TestSoapEndpoint(APITestCase):
+    def test_routing(self):
+        """Check that routing for Sigmax is active and correct"""
+        response = self.client.get(SOAP_ENDPOINT)
+        self.assertNotEqual(response.status_code, 404)
+
+    def test_http_verbs(self):
+        """Check that the SOAP endpoint only accepts POST and OPTIONS"""
+        not_allowed = ['GET', 'PUT', 'PATCH', 'DELETE', 'HEAD']
+        allowed = ['POST', 'OPTIONS']
+
+        superuser = SuperUserFactory.create()
+        self.client.force_authenticate(user=superuser)
+
+        for verb in not_allowed:
+            method = getattr(self.client, verb.lower())
+            response = method(SOAP_ENDPOINT)
+            self.assertEqual(response.status_code, 405)
```

Comment:
When you've an assertion in a loop it's more convenient to pass the reason as well. Like https://github.com/Amsterdam/signals/blob/master/api/app/tests/apps/signals/test_api_endpoints.py#L59